### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by atharvachavan2000**
Button sizes were mismatched in tablet and smaller views. Adjusted those. Also added margin to make the buttons look aesthetically good.

### CODE REVIEW

**Comment by atharvachavan2000** [comment]
Resolved issue #16  and added the font awesome kit to the contact page. Added a link to the homepage.

**Review by SuvamPrasd — APPROVED** [review]

**Comment by atharvachavan2000** [comment]
Added Marquee tag as per the request in the Slack channel.

File: param1.css [hunk 1]
```diff
@@ -8,6 +8,14 @@
 width: 100px;
 }
 
+h1{
```
Comment (SuvamPrasd) [inline param1.css:11]: You can give this style inside the h1 element only.

**Review by SuvamPrasd — CHANGES_REQUESTED** [review]
put the h1 style inside the h1 element itself.

**Pull request merged**